include/RAJA/pattern/launch/launch_core.hpp

```diff
@@ -297,21 +332,44 @@ launch(RAJA::resources::Resource res, LaunchParams const &params, const char *ke
     place = RAJA::ExecPlace::DEVICE;
   }
 
+  //
```

Comment:
looking up a few lines, can this impl just call the ExecPlace interface?

Reply:
Also, is the ::v1 necessary, isn't the namespace inline?

Reply:
> looking up a few lines, can this impl just call the ExecPlace interface?

We probably can, I think the launch exec methods would just all have to take a resource and we would have to create a proper resource if one is not provided at this interface.

Reply:
That's easy and probably a good change anyway to reduce redundancy in the exec methods.